src/python/pants/engine/scheduler.py

```diff
@@ -244,6 +244,9 @@ def invalidate_files(self, direct_filenames):
     def invalidate_all_files(self):
         return self._native.lib.graph_invalidate_all_paths(self._scheduler)
 
+    def check_invalidation_watcher_liveness(self):
```

Comment:
Type hints appreciated if you know what this returns. You'll need to wrap the next line in something like 

```python
return cast(bool, self._native.lib.check_invalidation_watcher_liveness(self._scheduler)
```

The cast tells MyPy "trust me on this one".